avm/res/storage/storage-account/main.bicep

```diff
@@ -176,6 +173,9 @@ param customerManagedKey customerManagedKeyType
 @description('Optional. The SAS expiration period. DD.HH:MM:SS.')
 param sasExpirationPeriod string = ''
 
+@description('Optional. The keyType to use with Queue & Table services. `Account` or `Service`.')
```

Comment:
If only those 2 are allowed, I'd suggest to remove them from the description into the 'allowed' set that also enables auto-complete :)

Reply:
Updated